spec/create-default-events.spec.js

```diff
@@ -0,0 +1,90 @@
+import $ from 'jquery'
+import rangy from 'rangy'
+
+import Cursor from '../src/cursor'
+import Editable from '../src/core'
+
+describe('Default Events', () => {
+  let $elem, editable, focus, blur
```

Comment:
You should write these onto `this` in mocha. This helps keeping test scopes separated.